src/yunohost/app.py

```diff
@@ -618,6 +619,7 @@ def app_upgrade(app=[], url=None, file=None, force=False):
         env_dict["YNH_APP_UPGRADE_TYPE"] = upgrade_type
         env_dict["YNH_APP_MANIFEST_VERSION"] = str(app_new_version)
         env_dict["YNH_APP_CURRENT_VERSION"] = str(app_current_version)
+        env_dict["YNH_APP_NO_BACKUP_UPGRADE"] = no_safety_backup
```

Comment:
The option name is `NO_BACKUP_UPGRADE`
https://github.com/YunoHost/yunohost/blob/dev/data/helpers.d/backup#L435

So you need to change here or in the helper line 435...
```suggestion
        env_dict["NO_BACKUP_UPGRADE"] = no_safety_backup
```

Reply:
Oops, sorry for that!